### PR TITLE
Notification upon download complete

### PR DESCRIPTION
**Pull request by kyteinsky**
Added notification popup (notify-send) to video/audio downloads to notify the user of the status of the download (successful or failed).

### CODE REVIEW

**Comment by kyteinsky** [comment]
There is an error from youtube-dl that pops up even upon successful download:
```
ERROR: '' is not a valid URL. Set --default-search "ytsearch" (or run  youtube-dl "ytsearch:" ) to search YouTube
```
But this does not concern this pull request much other than a download failed notification even upon successful download. This should be solved when that issue is solved in the youtube-dl repo.

**Comment by kyteinsky** [comment]
Solves #243

File: ytfzf [hunk 1]
```diff
@@ -1067,6 +1067,13 @@ open_player () {
 		else
 			youtube-dl -f "$video_pref"  "$@"  $YTFZF_SUBT_NAME || video_pref= open_player "$@"
 		fi
+
+    if [ $? -eq 0 ]; then
+        dl_status="Successful"
+    else
+        dl_status="Failed"
+    fi
+    notify-send "[ytfzf] Download $dl_status!" "$*"
```
Comment (Euro20179) [inline ytfzf:1076]: This should probably be after line 1068 so that $? refers to the right command.

Comment (kyteinsky) [reply ytfzf:1076]: Oh I didn't know that could be a problem. I just wanted to save some lines of code.

**Pull request merged**